src/library_pthread.js

```diff
@@ -1066,6 +1051,16 @@ var LibraryPThread = {
     // `runtimeKeepaliveCounter` to zero. Reset it now so the new thread won't
     // be affected.
     runtimeKeepaliveCounter = 0;
+
+#if isSymbolNeeded('$noExitRuntime')
+    // Same for noExitRuntime.  The default for pthreads should always be false
+    // otherwise pthreads would never complete and attempts to pthread_join to
+    // them would block forever.
+    // pthreads can still choose to set `noExitRuntime` explicitly, or
```

Comment:
```suggestion
    // Pthreads can still choose to set `noExitRuntime` explicitly, or
```
Unless there is a convention to never capitalize `pthread`?

Reply:
I tend not a capitalize it myself.